contracts/minters/open-edition-minter/src/helpers.rs

```diff
@@ -0,0 +1,118 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, BankMsg, Coin, coins, ContractInfoResponse, CustomQuery, Decimal, Deps, Empty, Event, Querier, QuerierWrapper, StdError, StdResult, to_binary, Uint128, WasmMsg, WasmQuery};
+use cw721_base::{Extension, MintMsg};
+use sg_std::{CosmosMsg, NATIVE_DENOM, Response, SubMsg};
+use sg_metadata::Metadata;
+use sg721::{ExecuteMsg as Sg721ExecuteMsg};
+
+use crate::ContractError;
+use crate::msg::{ConfigResponse, ExecuteMsg, QueryMsg};
+
+/// MinterContract is a wrapper around Addr that provides a lot of helpers
+/// for working with this.
+#[cw_serde]
+pub struct MinterContract(pub Addr);
+
+impl MinterContract {
+    pub fn addr(&self) -> Addr {
+        self.0.clone()
+    }
+
+    pub fn call<T: Into<ExecuteMsg>>(&self, msg: T) -> StdResult<CosmosMsg> {
+        let msg = to_binary(&msg.into())?;
+        Ok(WasmMsg::Execute {
+            contract_addr: self.addr().into(),
+            msg,
+            funds: vec![],
+        }
+        .into())
+    }
+
+    pub fn call_with_funds<T: Into<ExecuteMsg>>(
+        &self,
+        msg: T,
+        funds: Coin,
+    ) -> StdResult<CosmosMsg> {
+        let msg = to_binary(&msg.into())?;
+        Ok(WasmMsg::Execute {
+            contract_addr: self.addr().into(),
+            msg,
+            funds: vec![funds],
+        }
+        .into())
+    }
+
+    pub fn contract_info<Q, T, CQ>(&self, querier: &Q) -> StdResult<ContractInfoResponse>
+    where
+        Q: Querier,
+        T: Into<String>,
+        CQ: CustomQuery,
+    {
+        let query = WasmQuery::ContractInfo {
+            contract_addr: self.addr().into(),
+        }
+        .into();
+        let res: ContractInfoResponse = QuerierWrapper::<CQ>::new(querier).query(&query)?;
+        Ok(res)
+    }
+
+    pub fn config(&self, querier: &QuerierWrapper) -> StdResult<ConfigResponse> {
+        let res: ConfigResponse = querier.query_wasm_smart(self.addr(), &QueryMsg::Config {})?;
+        Ok(res)
+    }
+}
+
+pub fn dev_fee_msgs_and_amount(
```

Comment:
Is this being used anywhere?

Reply:
Yeah: https://github.com/racoon-supply-rac/launchpad/blob/0f299080a89a08988ecc3491117c8eb644f28891/contracts/minters/open-edition-minter/src/contract.rs#L311

There a few util functions within the code that could be refactored to be more general and put elsewhere that's one of them. We can discuss those